src/content/glossary/index.md

```diff
@@ -123,6 +127,10 @@ A crosslink provides a summary of a shard's state. It's how [shard](#shard) chai
 
 ## D {#section-d}
 
+### DAG
+
+DAG stands for Directed Acyclic Graph and without going into too many details, it is a crucial file in ETH and ETC mining. DAG file size grows over time, exactly every 30.000 blocks (Ethereum) or every 100 hours to be precise.
```

Comment:
```suggestion
A Directed Acyclic Graph (DAG) is used to make the Ethash Proof-of-Work (PoW) ASIC resistant. The DAG dataset is a resource used when calculating the proof-of-work, and requires choosing subsets of a fixed resource dependent on the nonce and block header. A new DAG dataset is created every 30000 blocks (a 100 hour window, called an epoch), and is a few GB in size.
```
This means that calculating the PoW requires choosing subsets of a fixed resource dependent on the nonce and block header.